### ISSUE TITLE
k8s YAML parsing error causes unknown repository error

### ISSUE THREAD

**Issue by rarkins** [renovatebot/renovate#16615]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

Parsing error of k8s files causes the repo to abort.

### Relevant debug logs

<details><summary>Logs</summary>

```
{
  "level": 50,
  "err": {
    "name": "YAMLException",
    "reason": "end of the stream or a document separator is expected",
    "mark": {
      "name": null,
      "position": 59,
      "line": 1,
      "column": 0,
      "snippet": " 1 | {{if ne (((.Values.elastic).disa ...\n 2 | {{if eq (((.Values.elastic).sing ...\n-----^\n 3 | apiVersion: argoproj.io/v1alpha1\n 4 | kind: Application"
    },
    "message": "end of the stream or a document separator is expected (2:1)\n\n 1 | {{if ne (((.Values.elastic).disa ...\n 2 | {{if eq (((.Values.elastic).sing ...\n-----^\n 3 | apiVersion: argoproj.io/v1alpha1\n 4 | kind: Application",
    "stack": "YAMLException: end of the stream or a document separator is expected (2:1)\n\n 1 | {{if ne (((.Values.elastic).disa ...\n 2 | {{if eq (((.Values.elastic).sing ...\n-----^\n 3 | apiVersion: argoproj.io/v1alpha1\n 4 | kind: Application\n    at generateError (/home/ubuntu/renovateapp/node_modules/js-yaml/lib/loader.js:183:10)\n    at throwError (/home/ubuntu/renovateapp/node_modules/js-yaml/lib/loader.js:187:9)\n    at readDocument (/home/ubuntu/renovateapp/node_modules/js-yaml/lib/loader.js:1645:5)\n    at loadDocuments (/home/ubuntu/renovateapp/node_modules/js-yaml/lib/loader.js:1688:5)\n    at loadAll (/home/ubuntu/renovateapp/node_modules/js-yaml/lib/loader.js:1701:19)\n    at Object.extractPackageFile (/home/ubuntu/renovateapp/node_modules/renovate/dist/modules/manager/argocd/extract.js:45:47)\n    at extractPackageFile (/home/ubuntu/renovateapp/node_modules/renovate/dist/modules/manager/index.js:55:13)\n    at getManagerPackageFiles (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/extract/manager-files.js:40:64)\n    at async /home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/extract/index.js:40:30\n    at async Promise.all (index 0)\n    at async extractAllDependencies (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/extract/index.js:39:28)\n    at async extract (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/process/extract-update.js:74:24)\n    at async extractDependencies (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/process/index.js:80:30)\n    at async Object.renovateRepository (/home/ubuntu/renovateapp/node_modules/renovate/dist/workers/repository/index.js:39:56)\n    at async renovateRepository (/home/ubuntu/renovateapp/app/worker/index.js:309:26)\n    at async /home/ubuntu/renovateapp/app/worker/index.js:569:5"
  },
  "msg": "Repository has unknown error",
  "time": "2022-07-16T11:51:13.019Z"
}
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
@JamieMagee seems we need an extra try/catch somewhere around the new YAML parsing

**Comment by JamieMagee** [comment]
Looks like @viceice resolved this in #16598?

**Comment by viceice** [comment]
No, it's ArgoCD manager, fixing it now
